pkgs/top-level/python-packages.nix

```
@@ -6752,7 +6752,9 @@ in {
 
   protonup = callPackage ../development/python-modules/protonup { };
 
-  protonvpn-nm-lib = callPackage ../development/python-modules/protonvpn-nm-lib { };
+  protonvpn-nm-lib = callPackage ../development/python-modules/protonvpn-nm-lib {
+    systemd-pkgs = pkgs.systemd;
```

Is there a systemd python library? If not please keep the name.

---

Yes, there is a systemd library, with the same name

---

[This is the systemd Python library, FYI](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/python-modules/systemd/default.nix#L20)

---

```suggestion
    pkgs-systemd = pkgs.systemd;
```
I think that name would be better.

---

Done